### ISSUE TITLE
Update README with directions on using screen reader with iOS simulator

### ISSUE THREAD

**Issue by akr3081** [PublicisSapient/enable-a11y#158]
Directions should be added so devs who do not have access to an iPhone can still verify screen reader functionality.
